src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -268,7 +268,16 @@ export abstract class _MatAutocompleteTriggerBase
 
     if (this.panelOpen) {
       // Only emit if the panel was visible.
-      this.autocomplete.closed.emit();
+      // The `NgZone.onStable` always emits outside of the Angular zone,
+      // so all the subscriptions from `_subscribeToClosingActions()` are also outside of the Angular zone.
+      // We should manually run in Angular zone to update UI after panel closing.
+      if (NgZone.isInAngularZone()) {
```

Comment:
This check isn't necessary since nothing special will happen if you do `NgZone.run` while it's already in the zone.